Obtaining file:///home/ralfaro/Develop/PyGeoTess
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
